front/public/locales/en/operationalStudies/manageTrainSchedule.json

```
@@ -79,6 +79,7 @@
   "launchPathFinding": "Launch pathfinding",
   "launchSimulation": "Start",
   "manageVias": "Steps management",
+  "missing_pathSteps": "At least one pathstep wasn't found in the infrastructure",
```

We usually use CamelCase, even though I find snake case more readable, especially for translations :/
```suggestion
  "missingPathSteps": "At least one pathstep wasn't found in the infrastructure",
```

---

done